models/contact/src/index.ts

```diff
@@ -124,25 +123,30 @@ export function createModel (builder: Builder): void {
     component: contact.component.CreateOrganization
   })
 
-  builder.createDoc(workbench.class.Application, core.space.Model, {
-    label: contact.string.Contacts,
-    icon: contact.icon.Person,
-    hidden: false,
-    component: contact.component.Contacts
-  }, contact.app.Contacts)
+  builder.createDoc(
+    workbench.class.Application,
+    core.space.Model,
+    {
+      label: contact.string.Contacts,
+      icon: contact.icon.Person,
+      hidden: false,
+      component: contact.component.Contacts
+    },
+    contact.app.Contacts
+  )
 
   builder.createDoc(view.class.Viewlet, core.space.Model, {
     attachTo: contact.class.Contact,
     descriptor: view.viewlet.Table,
-    open: contact.component.EditContact,
+    open: contact.component.EditPerson,
```

Comment:
why? what about Organization?

Reply:
I think we must remove this unused param....

Reply:
If it is unused, let's remove it. Because it is confusing.